devTools/phpstan-baseline.neon

```diff
@@ -264,12 +264,24 @@ parameters:
 			count: 2
 			path: ../tests/phpunit/AutoReview/EnvVariableManipulation/EnvTestCasesProvider.php
 
+		-
+			message: '#^Unused Infection\\Tests\\AutoReview\\EnvVariableManipulation\\EnvTestCasesProvider\:\:envTestCaseTupleProvider$#'
+			identifier: shipmonk.deadMethod
+			count: 1
+			path: ../tests/phpunit/AutoReview/EnvVariableManipulation/EnvTestCasesProvider.php
+
```

Comment:
all issues added to baseline are now false-positived, reported https://github.com/shipmonk-rnd/dead-code-detector/issues/211

Reply:
this has been fixed, not relevant anymore